examples/white.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+import blinkt
+
+blinkt.set_clear_on_exit(False)
+blinkt.set_all(255, 255, 255)
```

Comment:
Suggestion: maye you could update `set_all.py` to take text colour names, so if it detects *two* arguments (file name and colour name) it acts accordingly.

In fact there's a Python module - colour - which could help you support all sorts of inputs: https://pypi.org/project/colour/

Reply:
One for another day!